db/migrate/20210824160127_update_notified_at_with_created_at.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateNotifiedAtWithCreatedAt < ActiveRecord::Migration[6.1]
+  def change
+    reversible do |dir|
+      dir.up do
+        PlanningApplication.all.map do |planning_application|
+          planning_application.validation_requests.map do |request|
```

Comment:
`each`, `map` is when you want something back (altered by whatever callback function you pass)